no-tech-for-apartheid.md

```diff
@@ -42,9 +42,10 @@ It is so easy to fall into despair and feel hopeless. But if we take action toge
 
 * **Outreach**: Message your coworkers in your team, org, and ERGs to get them involved too, so we can build a movement that’s ready for this moment. Check out some [template communications](#template-email-to-your-coworkers-org-ergs) we have below.
 
-* **Mobilize**: When our leadership refuses to acknowledge how tech workers are impacted by the current crisis, we will honor our colleagues ourselves. Organizing a vigil is an opportunity for us to come together to honor the passing of tech workers like Mai. Pick a date and an entrance to your office where you will gather. To organize a vigil at your office, see [Bit.ly/VigilHowTo](https://bit.ly/VigilHowToo). 
+* **Mobilize**: When our leadership refuses to acknowledge how tech workers are impacted by the current crisis, we will honor our colleagues ourselves. Organizing a vigil is an opportunity for us to come together to honor the passing of tech workers like Mai. Pick a date and an entrance to your office where you will gather. To organize a vigil at your office, see [Bit.ly/VigilHowTo](https://bit.ly/VigilHowTo). 
 
-* **Learn**: Educate yourself about the current crisis. Bring your coworkers to the many virtual teach-ins that Palestinian groups have been organizing.
+* **Learn**: Educate yourself about the current crisis. Bring your coworkers to the many virtual teach-ins that Palestinian groups have been organizing. Upcoming teach-ins: 
+    * [Tech Workers Teach In (virtual)](https://us06web.zoom.us/j/86353038651?pwd=EoipHDubY5uTzFA3BIDe0du1iTAmc1.1): Tuesday 1pm PST
```

Comment:
Is "Tech Workers Teach In" a good description of this event? Seems, uh, very vague. Teach in about what? By tech workers? For tech workers? Both?

Also that link goes directly to the meeting ... should we have a registration to be able to contact people (name, email, phone, city, state, country/region, company, metro)? See http://bit.ly/NOTAMassCalls.

Reply:
Update the link with your bit.ly. I think the text is pretty good as-is, though. It's a teach-in about the crisis, by Palestinian groups, for tech workers. Is there something that you think could be more clear?

Reply:
I won't fret about it, but I'd like more specifics in the bullet line. If I don't read the whole "Learn" sentence, "Tech Workers Teach In" doesn't say much. Is it by or for tech workers? What is it actually about? (My understanding is this action is specifically about APEC?) & what Palestinian groups?